JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
-This demo illustrates the Popup component. Click the Details button under an employee picture to see what the Popup looks like. Here is how you can configure its main elements:
+This demo shows how to show and hide the Popup component, how to fill it with content, specify its position and other configurations. To get started with the DevExtreme Popup component, refer to the following tutorial for step-by-step instructions: [Getting Started with Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/).
```

Comment:
- "shows how to show" - I would try not to repeat similar words close together. 
- "fill it with content" - the word "populate" is a better choice in this context. 
- "specify configurations" - the word "configuration" implies a combination of multiple settings. In this configuration that control behaves like this, and in this configuration it behaves like that... It's the end result of setting individual options. I would use the word "options" or "settings" instead. 

I also don't like the second sentence being in the same paragraph. As I see it, this is a separate thought. The main thought of the first paragraphs is "this is what this demo module shows you". And then you immediately say "you can go to another topic". Should I? Or is it just a note that I can? I assume it is the second option - you may want to go there if you need some step-by-step instructions. But you can also continue with this demo module. This means that I would separate those two thoughts into individual paragraphs. 

```suggestion
This example demonstrates how to show and hide the Popup component, populate it with content, specify its position and other settings. 

To see step-by-step instructions on how to get started with the DevExtreme Popup component, refer to the following tutorial: [Getting Started with Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/).
```